src/editors/containers/ProblemEditor/components/SelectTypeModal/content/Preview.jsx

```diff
@@ -6,7 +6,7 @@ import {
   useIntl,
 } from '@edx/frontend-platform/i18n';
 import messages from './messages';
-import { ProblemTypes } from '../../../../../data/constants/problem';
+import { ProblemTypes, getProblemTypes } from '../../../../../data/constants/problem';
```

Comment:
Please update these to this new format as you go:
```suggestion
import { ProblemTypes, getProblemTypes } from '@src/editors/data/constants/problem';
```

Reply:
Fixed